### ISSUE TITLE
Add typescript GraphQL Codegen example?

### ISSUE THREAD

**Issue by RobRoseKnows** [Shopify/example-app--qr-code--remix#21]
I know Issue #1 was requesting a TypeScript version but I thought I'd reup that I'd like to see a version of this example fully typed, including with the GraphQL codegen as detailed [here](https://shopify.dev/docs/api/shopify-app-remix/v2/guide-graphql-types).

**Comment by vaclavek-kamil** [comment]
Yeah. I somehow got it working as it is now but I don't know what I did and can't replicate it now :d
